cmd/influxd/run.go

```diff
@@ -187,6 +188,31 @@ func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Ser
 			}
 		}
 
+		// Spin up any OpenTSDB servers
+		if config.OpenTSDB.Enabled {
```

Comment:
@otoolep sorry, not sollowing. This is at the same level as all the other inputs,and inside the if s != nil. If it needs moving, you'll need to be more explicit.